Marlin/src/lcd/e3v2/proui/dwinui.cpp

```diff
@@ -181,7 +181,7 @@ void DWINUI::drawString(uint16_t color, const char * const string, uint16_t rlim
 //  iNum: Number of digits
 //  x/y: Upper-left coordinate
 //  value: Integer value
-void DWINUI::drawInt(uint8_t bShow, bool signedMode, fontid_t fid, uint16_t color, uint16_t bColor, uint8_t iNum, uint16_t x, uint16_t y, int32_t value) {
+void DWINUI::drawInt(uint8_t bShow, bool signedMode, fontid_t fid, uint16_t color, uint16_t bColor, uint8_t iNum, uint16_t x, uint16_t y, long value) {
```

Comment:
Is a `long` of undetermined size required here? By using types with fixed bit lengths we avoid that ambiguity and never have to deal with any 64-bit `long` type, now or in future.

Reply:
hm. well... I suppose we can guarantee 32-bit with `int32_t`. I did a quick look through and looks to me most are calling for 16. I figured since the rest of the inline are cast to `long` they all should be. what do you think?